src/main/kotlin/net/irisshaders/lilybot/utils/_Utils.kt

```diff
@@ -0,0 +1,81 @@
+ * null check it, and respond accordingly. It takes a string input of the column that is requested and gets it, if it's
+ * null a private error message (i.e. for commands staff run), and null is returned. If null this should be handled with
+ * an elvis operator to return the action.
+ *
+ * @param inputColumn The column you wish to get from the database
+ * @return The column value from the database or null
+ * @author NoComment1105
+ */
+suspend fun EphemeralSlashCommandContext<*>.getFromConfigPrivateResponse(inputColumn: String) =
+	DatabaseHelper.selectInConfig(guild!!.id, inputColumn) ?: run {
+		respond {
+			content = "**Error:** Unable to access config for this guild! Is your configuration set?"
+		}
+		null
+}
+
+
+/**
+ * This is a simple function to get a value from the configuration database in an ephemeral slash command context,
+ * null check it, and respond accordingly. It takes a string input of the column that is requested and gets it, if it's
+ * null a public error message (i.e. for commands the people run), and null is returned. If null this should be handled
+ * with an elvis operator to return the action.
+ *
+ * @param inputColumn The column you wish to get from the database
+ * @return The column value from the database or a public error and null
+ * @author NoComment1105
+ */
+suspend fun EphemeralSlashCommandContext<*>.getFromConfigPublicResponse(inputColumn: String) =
+	DatabaseHelper.selectInConfig(guild!!.id, inputColumn) ?: run {
+		respond {
+			content = "**Error:** Unable to access config for this guild! Please inform a member of staff!"
+		}
+		null
+}
+
+/**
+ * This is a simple function to get a value from the configuration database in an ephemeral message command context,
+ * null check it, and respond accordingly. It takes a string input of the column that is requested and gets it, if it's
+ * null a private error message (i.e. for commands staff run), and null is returned. If null this should be handled with
+ * an elvis operator to return the action.
+ *
+ * @param inputColumn The column you wish to get from the database
+ * @return The column value from the database or a private error and null
+ * @author NoComment1105
+ */
+suspend fun EphemeralMessageCommandContext.getFromConfigPublicResponse(inputColumn: String) =
+	DatabaseHelper.selectInConfig(guild!!.id, inputColumn) ?: run {
+		respond {
+			content = "**Error:** Unable to access config for this guild! Please inform a member of staff!"
+		}
+		null
+}
+
+/**
+ * This is a simple function to check if the channel a command is being run in is a thread or not. If not, it informs
+ * the user. It should be handled with an if statement to return the action, depending on your desired response
+ *
+ * @return Boolean. True if the channel is a thread, false if not
+ * @author NoComment1105
+ */
+suspend fun EphemeralSlashCommandContext<*>.isThread(): Boolean =
```

Comment:
I hate to do this but Kord has an isInThread() check. I guess this is slightly neater because it has a response but like are people really dumb enough to need a response for that.

Reply:
I was not aware of this